subsys/fw_metadata/Kconfig

```diff
@@ -5,8 +5,8 @@
 #
 
 menuconfig FW_METADATA
-	bool
-	prompt "Firmware Metadata"
+	bool "Firmware Metadata"
+	default y
```

Comment:
But its not needed by builds not using B0, right? Isn't it quite intrusive to add this to builds where the data is never used?

Just the work needed to debug why this is even included is enough reason to not have it for non B0 builds IMO.

Reply:
I agree, it's sort of unrelated and risky, so I removed it for this PR.